werkomgeving/gegeven/index.md

```diff
@@ -20,29 +20,29 @@ In bovenstaande voorbeelden wordt met [Jan] de verwijzing bedoeld naar een speci
 
 ## Een gegevenstype
 
-Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten:
+Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten: 
 
-1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij?
+1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij? 
 2. Over welke eigenschap gaat zo'n gegeven? (dwz: relatie naar het CIM waarin de eigenschappen zijn beschreven)
-3. Welke waarden zijn mogelijk bij dit gegeven, welke beperkingen gelden hierbij? En hoeveel waarden zijn (gelijktijdig) mogelijk voor de betreffende eigenschap?
```

Comment:
Zou het ook kunnen dat de omstandigheden waarin een eigenschap zich bevindt bepalend is voor de waarden die voor een gegeven mogelijk zijn? Bijv. als een ander gegeven waarde ‘a’ heeft dan kan dit gegeven niet waarde ‘z’ hebben. Of sterker nog dat een bepaalde waarde voor een ander gegevens bepalend is voor het wel of niet voorkomen van een ander gegeven. Dus eigenlijk constraints die gelden.

Reply:
Ik denk van wel.